insights/src/main/resources/schema/insights.xsd

```diff
@@ -0,0 +1,9 @@
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
```

Comment:
We have a standard naming scheme for the subsystem config xsd files we ship. This file should be named:

wildfly-insights_1_0.xsd

Since this won't actually end up in the WildFly dist, this is fine:

jboss-eap-insights_1_0.xsd